lib/multiplexer/multiplexer.go

```diff
@@ -386,6 +387,8 @@ func isHTTP(in []byte) bool {
 	return false
 }
 
+var proxyHelloSig = []byte(sshutils.ProxyHelloSignature)
```

Comment:
nit: move near the other `fooPrefix`, maybe rename to `proxyHelloPrefix`?

Reply:
3f97ad2b3